rviz_visual_testing_framework/include/rviz_visual_testing_framework/visual_test_fixture.hpp

```diff
@@ -46,14 +46,23 @@
 #include "rviz_visual_testing_framework/internal/visual_test.hpp"
 #include "rviz_visual_testing_framework/page_objects/page_object_with_window.hpp"
 
+#include <ament_index_cpp/get_package_share_directory.hpp>
+#include <ament_index_cpp/get_package_prefix.hpp>
```

Comment:
Please alphabetically order includes.

Reply:
This still needs to be done.

Reply:
Noted. Changes made to this header file have been reverted.